VideoApi/VideoApi.Events/Handlers/TransferEventHandler.cs

```diff
@@ -47,8 +41,8 @@ protected override async Task PublishStatusAsync(CallbackEvent callbackEvent)
                 return;
             }
 
-            var roomQuery = new GetRoomByIdQuery(SourceConference.Id, callbackEvent.TransferredFromRoomLabel);
-            var room = await QueryHandler.Handle<GetRoomByIdQuery, Room>(roomQuery);
+            var roomQuery = new GetConsultationRoomByIdQuery(SourceConference.Id, callbackEvent.TransferredFromRoomLabel);
```

Comment:
is this command and handler a duplicate of the 1 i commented on earlier?

Reply:
It is, there is a strong overlap at the moment. The differences will start to arise once the interpreter consultation stories begin